### DISCUSSION TITLE
Sporadic behaviour

### DISCUSSION THREAD

**Discussion by junoslukan** [seedvault-app/seedvault#226] [Q&A]
Hi everyone,
I am using GrapheneOS and I appreciate Seedvault very much as it's just the app I need to seamlessly backup my data. I have had some trouble using it, however.
I am using Seedvault v11-1.0 with a Nextcloud storage provider.

When I manually trigger a backup (by using "Backup now"), most of the time I get "Backup failed" after a couple of seconds. 
I thought it might be related to unstable WiFi connection or battery level, but it's difficult to troubleshoot this: is there a log available somewhere? Or is looking at the logcat via adb the only option to figure out what's wrong?

Despite this issue, some apps do get backed up automatically, but again it seems completely random. There are a few that have been backed up recently, but most have not, and this seems to be independent from the "Exclude apps" status. 
Some excluded apps have been backed up more recently than some I didn't want to exclude.
Also, some other excluded apps show "Waiting to back up..." which I understand it means they have never been backed up. But if I excluded them, should they be waiting for it?

Does excluding apps work at all?

**Comment by lazytownfan** [comment]
Though I don't use NextcIoud, I can try to help for the other points.

> Also, some other excluded apps show "Waiting to back up..." which I understand it means they have never been backed up.

I sometimes observe this for "system"-like backup components. For example, I observe this in the phone call history and SMS messages, since I haven't used a SIM card in my GrapheneOS (on Android 12) device yet.

However, for me, Seedvault is pretty clear and not ambiguous on which apps successfully backed up or not - I don't recall an installed app (from F-Droid or Aurora Store) ever giving a status of "Waiting to back up...", though YMMV.

As of October 2021, going through System > Backup > Backup status in the Settings app will take you to which apps were eligible or not for Seedvault backups; and then which of the eligible apps were successfully backed up or not.
